test-ci/CMakeLists.txt

```diff
@@ -0,0 +1,21 @@
+# This is the project CMakeLists.txt file for the test subproject
+cmake_minimum_required(VERSION 3.16)
+
+# Include the components directory of the main application:
+#
+set(EXTRA_COMPONENT_DIRS "../components")
+
+# Set the components to include the tests for.
+# This can be overriden from CMake cache:
+# - when invoking CMake directly: cmake -D TEST_COMPONENTS="xxxxx" ..
+# - when using idf.py: idf.py -T xxxxx build
+#
+set(TEST_COMPONENTS "bm1397 stratum" CACHE STRING "List of components to test")
+
+include($ENV{IDF_PATH}/tools/cmake/project.cmake)
+
+set(IDF_TARGET "esp32s3")
+
+idf_build_set_property(COMPILE_DEFINITIONS "-DCONFIG_ASIC_FREQUENCY=100" APPEND)
+
+project(unit_test_stratum)
```

Comment:
This seems to be really close to `test/CMakeLists.txt`. To prevent forgetting to include future changes to `test/CMakeLists.txt` to `test-ci/CMakeLists.txt` maybe we can use CMake's `include`?

For example, if someone adds a new component to `TEST_COMPONENTS` in test/CMakeLists.txt but forgets to add it to test-ci/CMakeListst.txt, then the qemu focused unit test build might run into issues or may omit some tests?

Reply:
I added a link for now.